modules/tinymce/changelog.txt

```diff
@@ -1,31 +1,31 @@
 Version 5.5.0 (TBD)
     Added a TypeScript declaration file to the bundle output for TinyMCE core #TINY-3785
-    Added new `table_column_resizing` setting to control how the table is affected when resizing columns using the resize bars #TINY-6001
-    Added ability to load TinyMCE in inline mode within a ShadowRoot #TINY-6128
+    Added new `table_column_resizing` setting to control how columns are resized when resizing tables using the resize bars #TINY-6001
+    Added the ability to load TinyMCE in inline mode within a ShadowRoot #TINY-6128
     Added the ability to remove images on a failed upload using the `images_upload_handler` failure callback #TINY-6011
     Added `hasPlugin` function to the editor API to determine if a plugin exists or not #TINY-766
-    Added new `ToggleToolbarDrawer` command and query state handler to allow the toolbar drawer to be programmatically toggled and its toggle state to be checked #TINY-6032
+    Added new `ToggleToolbarDrawer` command and query state handler to allow the toolbar drawer to be programmatically toggled and the toggle state to be checked #TINY-6032
     Added the ability to delete and navigate HTML media elements without the media plugin #TINY-4211
     Added `fullscreen_native` setting to the fullscreen plugin to enable use of the entire monitor #TINY-6284
-    Added table related oxide variables to the Style API for more granular control over table cell selection appearance #TINY-6311
-    Added new `toolbar_persist` setting to control the visibility of the inline toolbar. #TINY-4847
-    Added new APIs to allow for programmatic control of the inline toolbar visibility. #TINY-4847
+    Added table related oxide variables to the internal style sheets for more granular control over table cell selection appearance #TINY-6311
```

Comment:
This is incorrect, `Style API` was correct/what this is normally referred to. It's the skin tool set of variables that allow for tweaking the less files.

Reply:
```suggestion
    Added table related oxide variables to the Style API for more granular control over table cell selection appearance #TINY-6311
```